src/main/java/duke/Event.java

```diff
@@ -0,0 +1,32 @@
+package duke;
+
+import java.time.LocalDate;
+/**
+ * Represents a Task with a specified start and end time.
+ */
+public class Event extends Task{
+    LocalDate fromdate;
+    LocalDate todate;
```

Comment:
```suggestion
    LocalDate fromDate;
    LocalDate toDate;
```

variable names should be in camelCase?